lib/modules/webserver/server.js

```diff
@@ -33,22 +31,18 @@ class Server {
     const main = serveStatic(this.buildDir, {'index': ['index.html', 'index.htm']});
 
     this.app = express();
-    expressWebSocket(this.app);
+    const expressWs = expressWebSocket(this.app);
 
     this.app.use(main);
     this.app.use('/coverage', coverage);
     this.app.use(coverageStyle);
 
-    this.app.ws('/', function(ws, _req) {
-      self.events.on('outputDone', () => {
-        if (ws.readyState === WEB_SOCKET_STATE_OPEN) {
-          return ws.send('outputDone');
-        }
-        // if the socket wasn't yet opened, listen for the 'open' event,
-        // then send the 'outputDone' data
-        ws.addEventListener('open', _event => {
-          ws.send('outputDone');
-        });
+    this.app.ws('/', (_ws, _req) => {});
+    const wss = expressWs.getWss('/');
```

Comment:
is this really right? if feels kinda off? (lines 40 and 41)

Reply:
Yes I believe it is, you can see official example here:
https://github.com/HenningM/express-ws/blob/master/examples/broadcast.js

L40 -> define the endpoint
L41 -> get the web socket to broadcast